ghmirror/data_structures/monostate.py

```diff
@@ -47,17 +47,24 @@
 
 class _GithubStatus:
 
-    SLEEP_TIME = int(os.environ.get("GITHUB_STATUS_SLEEP_TIME", 1))
-
-    def __init__(self):
+    def __init__(self, sleep_time):
+        self.sleep_time = sleep_time
         self.online = True
-        theard = threading.Thread(target=self.check)
-        theard.start()
+        thread = threading.Thread(target=self.check, daemon=True)
```

Comment:
please add in the comment or in the git commit message why we're using `daemon=True`

Reply:
added in https://github.com/app-sre/github-mirror/pull/103/commits/bf262d6167a375965d8ab883152700b356ac7285